src/syntax-interpreter.js

```diff
@@ -272,3 +286,9 @@ export class SyntaxInterpreter {
     return instruction;
   }
 }
+
+Object.defineProperty(SyntaxInterpreter.prototype, 'one-way', {
```

Comment:
I think `SyntaxInterpreter.prototype['one-way'] = SyntaxInterpreter.prototype['to-view'];` would be ok here.

Reply:
Will fix. I was also thinking of log deprecated in one-time and apply arguments to to-view, do you think its a bad idea ?